test/Microsoft.ML.Benchmarks/Numeric/Ranking.cs

```diff
@@ -21,8 +21,8 @@ public class RankingTrain
         [GlobalSetup]
         public void SetupTrainingSpeedTests()
         {
-            _mslrWeb10k_Validate = Path.GetFullPath(TestDatasets.MSLRWeb.validFilename);
-            _mslrWeb10k_Train = Path.GetFullPath(TestDatasets.MSLRWeb.trainFilename);
+            _mslrWeb10k_Validate = TestFramework.BaseTestClass.GetDataPath(TestDatasets.MSLRWeb.validFilename);
```

Comment:
Can't this just be `BaseTestClass.GetDataPath`? Why do you need to qualify it with `TestFramwork.`?

Reply:
It was done to just avoid adding a reference to ```using Microsoft.ML.Runtests``` statement.
i dont have a preference. so should i go with ```BaseTestClass.GetDataPath``` ?